scripts/ci/generate-changelog

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+set -ex
+
+GIT_TAGS=$(git tag -l --sort=-version:refname)
```

Comment:
I tried it locally by switching to an old tag and running this command from there and I got all tags as a result. If I understand this script correctly we will always generate the changelog between the latest two.

E.g. I do a tag 1.10.5 => I want a changelog for the commits between 1.10.4 and 1.10.5. This script will deliver me the changelog for 1.11.1 and 1.11.2, which is unexpected and a show stopper to me

Reply:
Hey thank for point this out! We changed this quite a bit, and I tested locally and on Jenkins. Seems to be working well. Let me know if any questions or issues with it 👍